pemutil/pem.go

```diff
@@ -740,3 +740,38 @@ func BundleCertificate(bundlePEM []byte, certsPEM ...[]byte) ([]byte, bool, erro
 
 	return bundlePEM, modified, nil
 }
+
+// UnbundleCertificate removes PEM-encoded certificates from a PEM-encoded
+// certificate bundle.
+func UnbundleCertificate(bundlePEM []byte, certsPEM ...[]byte) ([]byte, bool, error) {
```

Comment:
Optionally, return the original bundle if `len(certsPEM) == 0`

Reply:
8b54772